package section2_4.exception;

public class ExceptionTester1 {

	public static void main(String[] args) {
		

	}

}
